data/human/news.txt

```diff
@@ -669,17 +703,36 @@ news "dentist"
 			"If the galaxy has many alien species, I hope they all have teeth."
 			"The advent of almost perfect artificial teeth may tempt you to stop caring if your natural teeth start to rot. Trust me that taking care of your teeth saves a lot of pain from both your mouth and your wallet."
 			"I'm that one dentist that doesn't recommend anything."
+			"I used to work as a back-alley doctor before getting this job."
 		word
 			`"`
 
 
 
-news "epidemiologist"
+news "epidemiologist in human space"
```

Comment:
Good call to split these, going further we can develop these so that they say slightly different things (e.g. the epidemiologists on quarg worlds knowing more about their diseases).